From: Michael Davidsaver <mdavidsaver@bnl.gov>
Date: Thu, 12 May 2011 17:00:41 -0400
Subject: [PATCH] asyn: keep generated docs from being cleaned

---
 asyn/Makefile |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/asyn/Makefile b/asyn/Makefile
index 21382fb..29fd0f6 100644
--- a/asyn/Makefile
+++ b/asyn/Makefile
@@ -35,3 +35,5 @@ iocBoot_DEPEND_DIRS += testIPServerApp
 endif
 
 include $(TOP)/configure/RULES_TOP
+
+UNINSTALL_DIRS := $(filter-out $(INSTALL_HTML),$(UNINSTALL_DIRS))
-- 
