From: Michael Davidsaver <mdavidsaver@bnl.gov>
Date: Mon, 25 Mar 2013 18:22:54 -0400
Subject: [PATCH] mca: SIS doesn't build on RTEMS

---
 mca/mcaApp/SISSrc/Makefile |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/mca/mcaApp/SISSrc/Makefile b/mca/mcaApp/SISSrc/Makefile
index 77c934c..beeddb3 100644
--- a/mca/mcaApp/SISSrc/Makefile
+++ b/mca/mcaApp/SISSrc/Makefile
@@ -15,7 +15,7 @@ DBD += SIS38XXSupport.dbd
 # Build the library for ARCHs with VME capability.
 # This is vxWorks, RTEMS and Linux (with PCI/VME bridge)
 LIBRARY_IOC_vxWorks += SIS38XX
-LIBRARY_IOC_RTEMS += SIS38XX
+#LIBRARY_IOC_RTEMS += SIS38XX
 # If your version of Linux has support for VME devLib uncomment following line
 #LIBRARY_IOC_Linux += SIS38XX
 
@@ -33,7 +33,7 @@ SIS38XX_LIBS += $(EPICS_BASE_IOC_LIBS)
 
 #==================================
 PROD_IOC_vxWorks += SIS38XXTest
-PROD_IOC_RTEMS += SIS38XXTest
+#PROD_IOC_RTEMS += SIS38XXTest
 # If your version of Linux has support for VME devLib uncomment following line
 #PROD_IOC_Linux += SIS38XXTest
 
-- 
