From: Michael Davidsaver <mdavidsaver@bnl.gov>
Date: Tue, 25 Oct 2011 16:51:11 -0400
Subject: [PATCH] stream: Include aaiRecord aaoRecord

---
 stream/streamApp/Makefile |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/stream/streamApp/Makefile b/stream/streamApp/Makefile
index 15f6b9f..90aa08a 100644
--- a/stream/streamApp/Makefile
+++ b/stream/streamApp/Makefile
@@ -20,7 +20,7 @@ streamApp_DBD += base.dbd
 
 # In 3.14.12 aaiRecord.dbd and aaoRecord.dbd are part of base.dbd
 # In earlier versions, these records are buggy and not included by default
-#streamApp_DBD += aaiRecord.dbd aaoRecord.dbd
+streamApp_DBD += aaiRecord.dbd aaoRecord.dbd
 
 PROD_SRCS += streamApp_registerRecordDeviceDriver.cpp
 PROD_SRCS_DEFAULT = streamAppMain.cc
-- 
