From: Michael Davidsaver <mdavidsaver@bnl.gov>
Date: Wed, 27 Jul 2011 09:55:56 -0400
Subject: [PATCH] stream: patch20081007

---
 stream/src/StreamFormatConverter.cc |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/stream/src/StreamFormatConverter.cc b/stream/src/StreamFormatConverter.cc
index 30a8686..1d8577b 100644
--- a/stream/src/StreamFormatConverter.cc
+++ b/stream/src/StreamFormatConverter.cc
@@ -278,7 +278,7 @@ scanString(const StreamFormat& fmt, const char* input,
     if (*input == '\0')
     {
         // match empty string
-        value[0] = '\0';
+        if (value) value[0] = '\0';
         return 0;
     }
     if (fmt.flags & skip_flag)
-- 
