From: Michael Davidsaver <mdavidsaver@bnl.gov>
Date: Mon, 25 Mar 2013 10:54:50 -0400
Subject: [PATCH] stream: StreamDevice-2-6-patch20121113

---
 stream/src/AsynDriverInterface.cc |    5 ++++-
 1 files changed, 4 insertions(+), 1 deletions(-)

diff --git a/stream/src/AsynDriverInterface.cc b/stream/src/AsynDriverInterface.cc
index e362421..8f64180 100644
--- a/stream/src/AsynDriverInterface.cc
+++ b/stream/src/AsynDriverInterface.cc
@@ -618,9 +618,12 @@ writeHandler()
     asynStatus status;
     size_t written = 0;
 
+    pasynUser->timeout = 0;
+    if (pasynGpib)
+         pasynOctet->flush(pvtOctet, pasynUser);
+    else
     // discard any early input, but forward it to potential async records
     // thus do not use pasynOctet->flush()
-    pasynUser->timeout = 0;
     do {
         char buffer [256];
         size_t received = sizeof(buffer);
-- 
