From: Michael Davidsaver <mdavidsaver@bnl.gov>
Date: Wed, 8 Aug 2012 15:57:54 -0400
Subject: [PATCH] stream: fix async hang during reconnect.

---
 stream/src/AsynDriverInterface.cc |    5 +++++
 1 files changed, 5 insertions(+), 0 deletions(-)

diff --git a/stream/src/AsynDriverInterface.cc b/stream/src/AsynDriverInterface.cc
index b3a887f..a3794f8 100644
--- a/stream/src/AsynDriverInterface.cc
+++ b/stream/src/AsynDriverInterface.cc
@@ -762,6 +762,11 @@ readRequest(unsigned long replyTimeout_ms, unsigned long readTimeout_ms,
             clientName(), pasynUser->errorMessage);
         return false;
     }
+    else if (status != asynSuccess) {
+        // Not queued for some reason (eg. disconnected)
+        // try again later
+        startTimer(replyTimeout);
+    }
     // continues with:
     //    handleRequest() -> readHandler() -> readCallback()
     // or handleTimeout() -> readCallback(StreamIoTimeout)
-- 
